### PR TITLE
Sweets/collective

### PR DESCRIPTION
**Pull request by SweetmanTech**
# CRE8ORS Collective Claim Tickets
- Pay `0.8 ETH` to mint. 🎟️ 
- If connected wallet is `whitelisted`, show `Founders Pass`. ✅ 

Screenshot 2023-05-17 at 7 11 26 PM

### CODE REVIEW

File: components/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
-const LandingPage = () => (
-  <div className="h-screen overflow-y-auto text-black bg-white">
-    <SeoHead title="Cre8ors" description="Cre8ors" image="/CRE8ORSLOGO_ICON.svg" />
-    <Header />
-    <div className="flex flex-col items-center justify-around min-h-screen py-5">
-      {/* Add an empty div for spacing */}
-      <div />
-      <ClaimButton />
-      <Instructions />
+const LangingPage = () => {
```
Comment (crazzywizard) [inline components/LandingPage/LandingPage.tsx:8]: nit:should be LandingPage

File: components/LandingPage/Mobile/Mobile.tsx [hunk 2]
```diff
@@ -2,26 +2,24 @@ import Image from "next/image"
 import { Screen } from "../../../lib/constants"
 import Footer from "../../Footer"
 import ReserveClaimButton from "../../ReserveClaimButton"
+import TitleText from "../Desktop/Screens/ReserveScreen/TitleText"
+import PassImage from "../Desktop/Screens/ReserveScreen/PassImage"
+import Requirements from "../Desktop/Screens/ReserveScreen/Requirements"
```
Comment (crazzywizard) [inline components/LandingPage/Mobile/Mobile.tsx:7]: nit: this should be moved out of desktop folder structure.

File: components/PendingTransactionModal/MintButton.tsx [hunk 3]
```diff
@@ -1,38 +1,72 @@
...
+  const claimFoundersPass = async () => {
+    const maxQuantity = isTestnet ? 100 : 1
+    const userEntry = {
+      minter: address,
+      maxMint: maxQuantity,
+      mintPrice: 0,
+    }
+    const merkleProof = generateMerkleProof(userEntry)
+    const tx = await contract.purchasePresale(1, maxQuantity, 0, merkleProof)
+    await tx.wait()
```
Comment (crazzywizard) [inline components/PendingTransactionModal/MintButton.tsx:45]: lets surround contract calls with try/catch for logging purposes

**Review by crazzywizard — APPROVED** [review]
Minor Comments. LGTM!

File: components/PendingTransactionModal/MintButton.tsx (hunk 3 above)
Comment (SweetmanTech) [reply components/PendingTransactionModal/MintButton.tsx:45]: it's already surrounded by a `try / catch` right?

Is this not sufficient?

Screenshot 2023-05-19 at 3 01 09 PM

File: components/LandingPage/LandingPage.tsx (hunk 1 above)
Comment (SweetmanTech) [reply components/LandingPage/LandingPage.tsx]: good catch

Comment (SweetmanTech) [reply components/LandingPage/LandingPage.tsx]: updated ✅ 

Screenshot 2023-05-19 at 3 02 52 PM

**Pull request merged**